lib/membrane_live_web/controllers/webinar_controller.ex

```diff
@@ -44,7 +44,7 @@ defmodule MembraneLiveWeb.WebinarController do
          callback
        ) do
     with {:ok, webinar} <- Webinars.get_webinar(uuid),
-         {:ok, webinar_db} <- is_user_authorized(webinar, user_id) do
+         {:ok, webinar_db} <- is_user_authorized(webinar, user_id, callback) do
```

Comment:
I think we should compare here if this callback variable is `show/2`.

```suggestion
is_show_callback? <- callback == (&show_callback/2)
{:ok, webinar_db} <- is_user_authorized(webinar, user_id, is_show_callback?) do
```